docs/tools/bench.md

```diff
@@ -47,6 +47,7 @@ The current benchmarks are:
 * `quadratic`: The [quadratic formula][qf], including a hand-rolled implementation of square root.
 * `recfact`: Compute *n!* using recursive function calls.
 * `rectangles-area-difference`: Output the difference between the areas of rectangles (as a positive value) given their respective side lengths.
+* `rectangle fits inside`: Output whether or not a rectangle fits inside of another rectangle given the width and height lengths.
```

Comment:
Can you please use the same name as for the benchmark file? That would be `fitsinside`.

Reply:
Done! Sorry took me a couple days to see this.